models/prices/arbitrum/prices_arbitrum_tokens.sql

```diff
@@ -143,5 +143,7 @@ FROM
     ('bpet-xpettech-bpet','arbitrum','BPET',0x6daf586b7370b14163171544fca24abcc0862ac5, 18),
     ('sundae-sundae-token','arbitrum','SUNDAE',0x352f4bf396a7353a0877f99e99757e5d294df374, 18),
     ('gns-gains-network','arbitrum','GNS',0x18c11fd286c5ec11c3b683caa813b77f5163a122, 18),
-    ('gswift-gameswift','arbitrum','GSWIFT',0x580e933d90091b9ce380740e3a4a39c67eb85b4c, 18)
+    ('gswift-gameswift','arbitrum','GSWIFT',0x580e933d90091b9ce380740e3a4a39c67eb85b4c, 18),
+    ('ezeth-renzo-restaked-eth','arbitrum','EZETH',0x2416092f143378750bb29b79ed961ab195cceea5, 18),
+    ('rseth-rseth','arbitrum','EZETH',0x2416092f143378750bb29b79ed961ab195cceea5, 18)
```

Comment:
The last two rows are having the same address:
```suggestion
    ('ezeth-renzo-restaked-eth','arbitrum','EZETH',0x2416092f143378750bb29b79ed961ab195cceea5, 18)
```